python/ccxt/base/types.py

```diff
@@ -492,13 +493,31 @@ class LeverageTier:
     info: Dict[str, Any]
 
 
+class LedgerEntry:
+    id: Str
+    info: Any
+    timestamp: Int
+    datetime: Str
+    direction: Str
+    account: Str
+    referenceId: Str
+    referenceAccount: Str
+    type: Str
+    currency: Str
+    amount: Str
+    before: float
+    after: float
+    status: Str
+    fee: Fee
+
+
 FundingRates = Dict[Str, FundingRate]
 LastPrices = Dict[Str, LastPrice]
 Currencies = Dict[Str, CurrencyInterface]
 TradingFees = Dict[Str, TradingFeeInterface]
 IsolatedBorrowRates = Dict[Str, IsolatedBorrowRate]
 CrossBorrowRates = Dict[Str, CrossBorrowRate]
-TransferEntries = Dict[Str, TransferEntry]
+TransferEntries = List[TransferEntry]
```

Comment:
![image](https://github.com/user-attachments/assets/96af7a64-cc96-45ef-b6ba-78ebe09b286b)
it should a dict I think

Reply:
I'm obviously missing something somewhere. I'm trying to confirm the return of parseTransfers in the Exchange class which should return TransferEntries

It first goes through:
` this.filterByCurrencySinceLimit (result, code, since, limit)`
followed by:
`this.filterByValueSinceLimit`
followed by
`this.filterByLimit`
which I believe is returning an Array 

The return type of "filter_by_limit" in python/ccxt/base/exchange.py is also "List[Object]"

Assuming it is a dictionary, I was not able to determine the key used for the entries.

Reply:
@DelAnt Sorry you're right, it should be a list so the typescript type is wrong, and the C# one might need to be updated as well.